src/pages/in/embed/_id/button.vue

```diff
@@ -140,12 +141,25 @@ export default {
       }
     },
     async doLike() {
-      if (!this.isMaxLike) {
+      if (!this.isMaxLike && !this.isCreator) {
         this.like();
         logTrackerEvent(this, 'LikeButtonFlow', 'clickLike', 'clickLike(embed)', 1);
       } else if (this.canSuperLike) {
         await this.newSuperLike();
         await this.updateSuperLikeStatus();
+      } else {
+        if (this.ctaClickCount === undefined) {
+          this.ctaClickCount = 1;
+        } else {
+          this.ctaClickCount = (this.ctaClickCount + 1) % 5;
+          if (this.ctaClickCount !== 0) return;
```

Comment:
Should it return when `this.ctaClickCount` change from `undefined` to `1`?
If so, prefer some hacks like `this.ctaClickCount = (this.ctaClickCount || 0) + 1` to unify the logic

Reply:
should not return 1st and every 5th time